backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AnsibleCallback.java

```diff
@@ -27,6 +27,7 @@
 @Typed(AnsibleCallback.class)
 public class AnsibleCallback implements CommandCallback {
     private static final Logger log = LoggerFactory.getLogger(AnsibleCallback.class);
+    private static final int UNKNOWN_COUNTER_LIMIT = 180;
```

Comment:
please make it configurable and better to set time here rather than amount of iterations, in case the frequency would change